### PR TITLE
Typography re-work without using tables.

### LINKED ISSUE #790: Typography is not mobile friendly
Use the same approach as in other pages.

image

### PR DESCRIPTION
**Pull request by DieselTripleX**
## Description

closes #790 

Typography page was re-worked without using tables, which provides better mobile compatibility.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/docs/src/components/DocsCode.vue [hunk 1]
```diff
@@ -9,6 +9,20 @@ import 'prismjs/components/prism-bash'
 import VuePrismComponent from 'vue-prism-component'
 
 export default {
+  data() {
```
Comment (m0ksem) [inline packages/docs/src/components/DocsCode.vue:12]: Would be nice to have a comment on why exactly do you need to rerender VuePrismComponent on code prop change. Why not just pass code prop to VuePrismComponent?

**Review by m0ksem — CHANGES_REQUESTED** [review]

File: packages/docs/src/components/DocsCode.vue (hunk 1 above)
Comment (asvae) [reply packages/docs/src/components/DocsCode.vue:12]: Yeah, doesn't seem appropriate.

File: packages/docs/src/components/DocsCode.vue [hunk 2]
```diff
@@ -1,4 +1,5 @@
 <template>
+  <!-- Tricky way to update code rendering component via v-if and nextTick. Watch handler() code below -->
```
Comment (m0ksem) [inline packages/docs/src/components/DocsCode.vue:2]:
```suggestion
  <!-- Tricky way to update code rendering component via v-if and nextTick. See `code` prop watcher code below -->
```

File: packages/docs/src/components/DocsCode.vue [hunk 3]
```diff
@@ -18,7 +19,7 @@ export default {
     code: {
```
Comment (m0ksem) [inline packages/docs/src/components/DocsCode.vue:19]: Move `watch` under `props`.

Follow this order:
```
Props
Data
Computed
Methods
Watch
```

**Review by m0ksem — CHANGES_REQUESTED** [review]

File: packages/docs/src/components/DocsCode.vue (hunk 3 above)
Comment (DieselTripleX) [reply packages/docs/src/components/DocsCode.vue:19]: 👍

**Review by m0ksem — APPROVED** [review]

**Comment by m0ksem** [comment]
@DieselTripleX, resolve conflicts, please.

**Review by m0ksem — APPROVED** [review]

**Pull request merged**